mackup/applications/lftp.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = lftp
+
+[configuration_files]
+.config/lftp/rc
```

Comment:
You should leverage the XDG config folder, see https://github.com/lra/mackup/blob/master/mackup/applications/tig.cfg#L7-L8 for an example.

Reply:
OK! thanks @lra I made modifications as suggested…

Reply:
👍